src/main/scala/com/fulcrumgenomics/bam/EstimatePoolingFractions.scala

```diff
@@ -190,7 +189,7 @@ class EstimatePoolingFractions
     iterator.setIncludeNonPfReads(false)
     iterator.setMappingQualityScoreCutoff(this.minMappingQuality)
     iterator.setQualityScoreCutoff(this.minBaseQuality)
-    iterator.toIterator
+    javaIteratorAsScalaIterator(iterator)
   }
```

Comment:
That's not a bad idea.  It's also annoying that SamLocusIterator implements `Iterable` and `Iterator`, making the implicits ambiguous and this explicit call necessary.